querybook/webapp/components/ListDataDoc/components/DataDocsName.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const UntitledText = styled.span`
+    opacity: 0.7;
+    font-style: italic;
+`;
+
+const Link = styled.a`
```

Comment:
took me all the way here to realize Link is not the standard link component, but why? also using a would not work with React router so you need to use the standard link